jetbrains-rider/src/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetLambdaBuilder.kt

```diff
@@ -24,18 +34,37 @@ class DotNetLambdaBuilder : LambdaBuilder() {
         samOptions: SamOptions,
         onStart: (ProcessHandler) -> Unit
     ): CompletionStage<BuiltLambda> {
-
+        val project = module.project
         val customTemplate = FileUtil.createTempFile("template", ".yaml", true)
         val logicalId = "Function"
+
         SamTemplateUtils.writeDummySamTemplate(
             tempFile = customTemplate,
             logicalId = logicalId,
             runtime = runtime,
-            codeUri = "",
+            codeUri = getCodeUriPath(project, handler),
```

Comment:
We use this code path if the project does not have a template, in that case codeUri should point to root of where we need to build from.

Reply:
Updated the logic to get a base project directory for a handler element.